exonum-java-binding/common/pom.xml

```diff
@@ -148,14 +154,43 @@
         <groupId>org.xolstice.maven.plugins</groupId>
         <artifactId>protobuf-maven-plugin</artifactId>
         <extensions>true</extensions>
+        <configuration>
+          <protocArtifact>com.google.protobuf:protoc:${protobuf.version}:exe:${os.detected.classifier}</protocArtifact>
+          <protoSourceRoot>${basedir}/src/main/proto/exonum/java</protoSourceRoot>
```

Comment:
Why is this needed? It recognizes the packages, the protoSourceRoot must remain the default.